src/ui-shell/ui-shell.stories.ts

```diff
@@ -50,6 +53,7 @@ storiesOf("UI Shell", module)
 			</ibm-header>
 		`,
 		props: {
+			hasHamburger: select("Show Hamburger", { "Yes": true, "No": false }, true),
```

Comment:
I belive this could be simplified with the `boolean` knob: https://github.com/storybooks/storybook/tree/next/addons/knobs#boolean

something like
```suggestion
			hasHamburger: boolean("Show Hamburger", true),
```

Reply:
Great idea! Haven't used this Storybook addon yet... guess I should spend some more time with the docs ;)